janitor/functions.py

```diff
@@ -1198,9 +1198,23 @@ def change_type(df, column: str, dtype):
     :param df: A pandas dataframe.
     :param column: A column in the dataframe.
     :param dtype: The datatype to convert to. Should be one of the standard
+    :param ignore_exception: {False, "fillna", "keep_values"}.
```

Comment:
@mdini, I think this line should go below the one below it. Could you please change that?